www/include/core/autologout/autologout.php

```diff
@@ -25,15 +25,59 @@ function change_status(xhr2) {
 	var docXML= xhr2.responseXML;
 	var items_state = docXML.getElementsByTagName("state");
 	var items_time = docXML.getElementsByTagName("time");
+    var timezoneItem = docXML.getElementsByTagName("timezone");
+    var realTimezone = timezoneItem.item(0).firstChild.data;
 	var state = items_state.item(0).firstChild.data;
 	var currentTime = items_time.item(0).firstChild.data;
 
+    // storing user's local timezone to the localStorage
+    localStorage.setItem('realTimezone', realTimezone);
+
 	if (state == "ok") {
 		if (document.getElementById('date')) {
-			document.getElementById('date').innerHTML = currentTime;
+			dateLocale(realTimezone, currentTime);
+
 		}
 	} else if (state == "nok") {
 		window.location.replace("./index.php");
 	}
 	setTimeout("check_session()", <?php echo $tM; ?>);
-}
\ No newline at end of file
+}
+
+//add the locale timezone and locale to the displayed hours in the header
+function dateLocale(timezone,currentTime){
+    var userLocale = localStorage.getItem('locale');
+    moment.locale(userLocale);
+    var userTime = parseInt(currentTime, 10);
+    var userDisplay = moment.unix(userTime).tz(timezone).locale(userLocale);
+    jQuery("#date").text(userDisplay.format('LLL'));
+}
+
+
+// use of moment() to format dates for each occurence
+function formatDateMoment() {
+
+    // get locale and GMT preferences from localStorage
+    var userTimezone = localStorage.getItem('realTimezone');
+    var userLocale = localStorage.getItem('locale');
+    moment.locale(userLocale);
+
+    //creating a collection of occurences
+    jQuery(".isTimestamp").each(function(index, element) {
+        var myElement = jQuery(element);
+
+        //forcing a numeric timestamp
+        var currentDate = parseInt(myElement.text()) * 1000;
+
+        //checking the choosen format
+        if (!isNaN(currentDate)){
```

Comment:
forgetting a white space in front of the bracket

Reply:
change done